internal/slicer/slicer.go

```diff
@@ -201,9 +212,14 @@ func Run(options *RunOptions) (*Report, error) {
 			if len(pathInfo.Arch) > 0 && !contains(pathInfo.Arch, arch) {
 				continue
 			}
-			if done[targetPath] || pathInfo.Kind == setup.CopyPath || pathInfo.Kind == setup.GlobPath {
+			if pathInfo.Kind == setup.CopyPath || pathInfo.Kind == setup.GlobPath {
 				continue
 			}
+			if done[targetPath] {
+				// The content created would have had the same properties. We
+				// are only adding the slice to the existing report entry.
+				report.Entries[targetPath].Slices[slice] = true
+			}
```

Comment:
[Note for reviewer] This is just so that the report contains all slices that declare a path that is not extracted from the package. It was found when written the test for the bugfix.